packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/navigation_callback_handler_delegate.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import 'webview_platform.dart';
+
+/// Interface for callbacks made by [NavigationCallbackHandlerDelegate].
+///
+/// The webview plugin implements this class, and passes an instance to the
+/// [NavigationCallbackHandlerDelegate].
+/// [NavigationCallbackHandlerDelegate] is notifying this handler on events that
+/// happened on the platform's webview.
+abstract class NavigationCallbackHandlerDelegate extends PlatformInterface {
```

Comment:
`WebViewControllerDelegate` should also have a method called `setNavigationCallbackHandler` that takes this as a parameter.

Reply:
Overlooked this one, thank you for pointing it out. Fixed in commit 8684ac3.